assets/chezmoi.io/docs/user-guide/frequently-asked-questions/encryption.md

```diff
@@ -100,3 +100,54 @@ $ chezmoi add --encrypt ~/.ssh/id_rsa
 When you run `chezmoi init` on a new machine you will be prompted to enter your
 passphrase once to decrypt `key.txt.age`. Your decrypted private key will be
 stored in `~/.config/chezmoi/key.txt`.
+
+## How to migrate between encryption tools?
+
+To migrate from gpg to age encryption or vise versa, one can follow these steps:
+
+1. Make sure you have all encrypted files applied (e.g. `chezmoi apply` decrypts files and places them in their destinations).
+2. Update chezmoi configuration to use the new encryption method
```

Comment:
Make sure that the text is wrapped. I don't know the typical max width, but it is shorter than this. Don't wrap the links, but it may mean that there's just one link per line.

Reply:
Good point. The rest of the document seems to aim at 80 characters. I'll do the same.